### PR TITLE
DEV2-2085: Log to output instead of dev tools

### PR DESCRIPTION
**Pull request by assapir**

### CODE REVIEW

File: src/utils/logger.ts [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+  // eslint-disable-next-line no-use-before-define
+  private static instance: Logger;
+
+  private outputChannel: OutputChannel;
+
+  private constructor() {
+    this.outputChannel = vscode.window.createOutputChannel("Tabnine");
+  }
+
+  public static log(message: string): void {
```
Comment (dimacodota) [inline src/utils/logger.ts:14]: consider to add a different log level output support (error/info)

Comment (assapir) [reply src/utils/logger.ts:14]: with a different prefix?

**Review by dimacodota — APPROVED** [review]
lgtm 🚀

File: src/utils/logger.ts (hunk 1 above)
Comment (assapir) [reply src/utils/logger.ts:14]: ok, cool. will do

**Pull request merged**